### ISSUE TITLE
Multiline disable for long list of disabled messages

### ISSUE THREAD

**Issue by Pierre-Sassoulas** [PyCQA/pylint#5258]
### Current problem

Sometime disable need to span multiple lines :

```python
# pylint: disable=invalid-name,too-many-public-methods,attribute-defined-outside-init
# pylint: disable= useless-object-inheritance,too-few-public-methods
```

Or contrived example with ``disable-next`` or inline comment (how many message can you want to disable on one line ?)
```python
# pylint: disable-next=invalid-name,too-many-public-methods,attribute-defined-outside-init, useless-object-inheritance,too-few-public-methods
actual_problem = 0
```

```python
actual_problem = 0 # pylint: disable=invalid-name,too-many-public-methods,attribute-defined-outside-init, useless-object-inheritance,too-few-public-methods
```

### Desired solution

It would be nice to be able to do:

```python
# pylint: disable=invalid-name,too-many-public-methods,attribute-defined-outside-init
# pylint: useless-object-inheritance,too-few-public-methods
```
or
```python
# pylint: disable-next=invalid-name,too-many-public-methods,attribute-defined-outside-init
# pylint: useless-object-inheritance,too-few-public-methods
actual_problem = 0
```
or
```python
actual_problem = 0 # pylint: disable=invalid-name,too-many-public-methods, attribute-defined-outside-init,
# pylint: useless-object-inheritance,too-few-public-methods
```

### Additional context

This is required in order to add enforced justification in disable : https://github.com/PyCQA/pylint/issues/5253
